codalab/server/bundle_manager.py

```diff
@@ -651,18 +660,17 @@ def _check_resource_failure(
                 return global_fail_string % (pretty_print(value), pretty_print(global_min))
         return None
 
-    def _get_staged_bundles_to_run(self):
+    def _get_staged_bundles_to_run(self, user_info_cache):
         """
         Fails bundles that request more resources than available for the given user.
         Note: allow more resources than available on any worker because new
         workers might get spun up in response to the presence of this run.
-
+        :param user_info_cache: a dictionary structured as {user id : user information} to track
```

Comment:
Why not use the same doc string as the other instance?

Reply:
Yeah, sure.